web/themes/custom/pineapple/components/_heading.twig

```diff
@@ -0,0 +1,10 @@
+{% set heading_attributes = heading_attributes|default(create_attribute()) %}
+{% set heading_level = heading_level|default('3') %}
```

Comment:
Okay i think you should put heading_level above of heading_attributes, the reason for this is because in other files there's a couple of places where you're basically adding the page title class, but because we know the h1 is always gonna be the page title, you can add the page title class to your heading attributes if the heading level is 1 (using a ternary operator of course)